photorepoapp-api/src/main/java/config/AmazonConfig.java

```diff
@@ -0,0 +1,26 @@
+package config;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AmazonConfig {
+
+    @Bean
+    public AmazonS3 s3() {
+        AWSCredentials awsCredentials = new BasicAWSCredentials(
+          "your access ket",
+          "your secret key"
+        );
```

Comment:
Look at uploading our API keys to a secrets section in our Github repo -- https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets